README.MD

```diff
@@ -80,4 +84,4 @@ OpenCV2 needs to be built from source according to the following [instructions]
 
 For basic example of functions, please see the ForgeScan/src/Examples to use ForgeScan
 
-For examples of ROS related uses, please call the run_reconstructions service from the scan_manager node.
\ No newline at end of file
+For examples of ROS related uses, please call the run_reconstructions service from the scan_manager node.
```

Comment:
What do you think about adding the specific ROS run/launch command (and the C++ commands)?
E.g., `ros2 run forgescan_realsense ...`

Are there files or other setup we need for the `scan_manager`?